platform/o.n.core/src/org/netbeans/beaninfo/editors/FontEditor.java

```diff
@@ -140,10 +136,7 @@ public void paintValue (Graphics g, Rectangle rectangle) {
     }
 
     private void paintText (Graphics g, Rectangle rectangle, String text) {
-        if( antialias && g instanceof Graphics2D ) {
-            ((Graphics2D)g).setRenderingHint(RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
-            ((Graphics2D)g).setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
-        }
+        GraphicsUtils.configureRenderingHints(g);
```

Comment:
Here's another spot where the new behavior might end up enabling subpixel antialiasing, vs. the old standard antialiasing. That's good.